.github/workflows/tests_doc.yml

```diff
@@ -13,18 +10,33 @@ defaults:
     shell: bash
 
 jobs:
+  job-skipper:
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v2.1.0
+        with:
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
+          paths: '["docs/**"]'
+
   test-doc:
+    needs: job-skipper
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
     steps:
     - uses: actions/checkout@v2
+      if: needs.job-skipper.outputs.should_skip != 'true'
     - name: Set up Python 3.9
       uses: actions/setup-python@v2
+      if: needs.job-skipper.outputs.should_skip != 'true'
       with:
         python-version: 3.9
     - name: Tests
+      if: needs.job-skipper.outputs.should_skip != 'true'
```

Comment:
Just for the note, it should be possible to move this condition at the job level (i.e. just after the ``needs: job-skipper`` line) to avoid duplicating it and starting this job at all during the run.

Reply:
Nice tip, I didn't know about that

Reply:
Basically, you can put the 'if' key almost everywhere, but not all the conditions are working everywhere (for example, test based on 'matrix.os' doesn't work at the job level, but only at the step one).